frontend/src/pages/acceleratorProfiles/screens/manage/ManageAcceleratorProfile.tsx

```diff
@@ -78,6 +95,7 @@ const ManageAcceleratorProfile: React.FC<ManageAcceleratorProfileProps> = ({
         <ManageAcceleratorProfileFooter
           state={state}
           existingAcceleratorProfile={existingAcceleratorProfile}
+          validK8sData={isK8sNameDescriptionDataValid(profileNameDesc)}
```

Comment:
Better off moving all form data validation into this component and passing a single valid boolean down to the footer for the enablement of the button.